pkg/cmd/cmd.go

```diff
@@ -37,6 +50,16 @@ func RunContext(ctx context.Context, name string, args ...string) (string, error
 		return stdout.String(), err
 	}
 
+	stdin := ctx.Value(stdin)
```

Comment:
I hate passing values via the context, but I don't know any better way without breaking the API for now...

Reply:
yeah, unfortunately we have the args already, so I couldn't use the options approach.
There's another way: we can convert args from `string` to interface type and then detect if the interface is an option, but that may be even worse than using the context :)

Reply:
yeah, context is better ;)

Reply:
or a separate method